src/PayID/pay-id-client-interface.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * An interface for a PayID client.
+ *
+ * @warning This class is experimental and should not be used in production applications.
+ */
+export default interface PayIDClient {
+  /**
+   * Retrieve the XRP Address authorized with a PayID.
```

Comment:
associated with a PayID?

Reply:
yup. fixed in `pay-id-client-ts` as well.